guides/Circle CI 2.md

```diff
@@ -0,0 +1,195 @@
+# Circle CI 2.0 configurations for Rails project
+
+## Basic Concepts
+
+CircleCI 2.0 operates on a differently structurized YAML file located under `.circleci/config.yml`.
+
+It's safe to say that every config should contain at least three keywords: `version, jobs`.
+Initially, it should start looking something like this:
+```
+---
+version: 2
+jobs:
+    build:
+        working_directory: ~/your-app-name
+    steps:
+        - checkout
+```
+This intital setup is not going to work. But it’s the basic shell of the config file we’ll need.
```

Comment:
This setup won't work, but it provides the framework of the configuration file. The remainder of the guide will explain how to flesh is out.